rotkehlchen/api/v1/schemas.py

```diff
@@ -1002,10 +997,9 @@ class StatisticsValueDistributionSchema(Schema):
     )
 
 
-class HistoryProcessingSchema(Schema):
+class HistoryProcessingSchema(AsyncQueryArgumentSchema):
```

Comment:
same as above. This is used here: 

https://github.com/rotki/rotki/blob/98f49cd3ed26c641fec03b78eff9fe1872385fbf/rotkehlchen/api/v1/resources.py#L974

Also wonder why tests did not catch this. Find out why.